### ISSUE TITLE
The border of some buttons has insufficient contrast with the background

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3551]
### Describe the Bug with repro steps

1. Create a new flow with designer standalone (nx serve).
2. Observe that border of "+" button to add an action has insufficient contrast with background.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/793c7669-95e3-4b8f-8a1a-109a6a82d6ec)

### Browser

Edge

### Additional context

Related internal bug: https://msazure.visualstudio.com/One/_workitems/edit/13416807

AB#25622192

**Comment by hartra344** [comment]
For the time being I'm darkening the borders here to make them stand out more and removing the non-hover box shadow so hover stands out more. That should bring us up to the minimum a11y spec.
